install.php

```diff
@@ -77,8 +77,7 @@
 				</p>
 			</div>
 		</div>
-		<div style="display: none;">
-			<PLAINTEXT>
```

Comment:
The main point of <plaintext> here was to stop html processing entirely, but if that's not a thing any more and `display:none` is the best we can get, <pre> seems a good a tag as any